### PR TITLE
enhancements

### PR DESCRIPTION
**Pull request by AHReccese**
#### Reference Issues/PRs
#27 
- add CLI tests to `test.yml`
- some refactorings applied
- Readme updated

### CODE REVIEW

**Comment by AHReccese** [comment]
@sadrasabouri 
Sadra could you please update the README links and badges?

**Comment by AHReccese** [comment]
If you can do it, let me know so I can assign this PR to you too to do pairing stuff!
@sadrasabouri

**Comment by sadrasabouri** [comment]
> @sadrasabouri Sadra could you please update the README links and badges?

Sure. I will do it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 ## [0.1] - 2024-05-29
 ### Added
+- `CLI` Handler
```
Comment (sadrasabouri) [inline CHANGELOG.md:12]: I will change it to `handler`, is that OK?

File: README.md [hunk 2]
```diff
@@ -35,7 +35,7 @@
 
 ## Overview
 <p align="justify">
-DMeta is an open source Python package that provides ... TODO
+DMeta is an open source Python package that removes metadata of microsoft office files.
```
Comment (sadrasabouri) [inline README.md:38]: I will change it to `... Microsoft Office ...`, is that OK?

File: README.md [hunk 3]
```diff
@@ -84,30 +84,68 @@ DMeta is an open source Python package that provides ... TODO
...
+clear_all()
+```
+#### Update metadata for a .docx file
+```python
+import os
+from dmeta.functions import update
+
+CONFIG_FILE_PATH = os.path.join(os.getcwd(), "CONFIG.json") 
+DOCX_FILE_PATH = os.path.join(os.getcwd(), "SAMPLE.docx")
+update(DOCX_FILE_PATH, DOCX_FILE_PATH)
```
Comment (sadrasabouri) [inline README.md:108]: One of them should be config, I will fix it.

File: README.md [hunk 4]
```diff
@@ -84,30 +84,68 @@ DMeta is an open source Python package that provides ... TODO
 - Run `pip install .`
 
 ## Usage
+### Programmatically
```
Comment (sadrasabouri) [inline README.md:87]: I suggest using `### In Python` instead.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Great work @AHReccese 
I have a minor comment. Plus that I would suggest to use small case words for file names instead, ex `sample.docx` instead of `SAMPLE.docx`.
I will apply some changes and do the needed changes in the repository and then you can either apply or ignore these suggestions.

File: CHANGELOG.md (hunk 1 above)
Comment (AHReccese) [reply CHANGELOG.md]: Yes.

File: README.md (hunk 2 above)
Comment (AHReccese) [reply README.md]: Sure, go ahead.

File: README.md (hunk 3 above)
Comment (AHReccese) [reply README.md]: shit, yes you are right, the first one I guess.

File: README.md (hunk 4 above)
Comment (AHReccese) [reply README.md:87]: Go ahead and apply yourself, then finally ping me, I will review them.

**Comment by AHReccese** [comment]
Great Job Sadra, Go ahead and apply them all, then tag me for a final review.

**Review by sadrasabouri — APPROVED** [review]
LGTM, @AHReccese please review. If it is OK, we can merge it.

**Comment by AHReccese** [comment]
Ok, great, lemmi review everything and then we will go for it.
@sadrasabouri 
I will take care of the rest.

**Pull request merged**